### PR TITLE
[PUB-1751] Remove duplicated packages in composer

### PR DESCRIPTION
**Pull request by anafilipadealmeida**
## Description
Remove dependencies that exist in the root package.json from the composer package.json file

## Context & Notes
We moved the composer to Publish, along with its package.json file filled with dependencies, including babel and plugins. We can’t have multiple versions of Babel in the repository and we want to update Babel to 7.

[PUB-1751](https://buffer.atlassian.net/browse/PUB-1751)

## Checklist

-   [ ] My code follows the code style and guidelines of this project. 
-   [ ] I have added tests to cover my changes.
-   [ ] All new and existing tests passed.
-   [ ] I have tested different user stories. 
-   [ ] I have identified similar or related features and tested they are still working.
-   [ ] I have performed a self-review of my own code.
-   [ ] I have tested my changes/additions in the latest Chrome, Firefox, and Safari.
-   [ ] I have commented my code, particularly in hard-to-understand areas.

#### Staging Deployment

To visit the URL of the staging deployment, please click "Show all checks" at the bottom of this PR and click "details" next to `bufferbotbrains/cicd-buffer-publish-legacy`

### CODE REVIEW

**Comment by anafilipadealmeida** [comment]
@hamstu The composer seems to be working fine with these changes, however I'd like to have someone else testing it 😅I'm just facing one problem: if I delete "@storybook/react": "3.1.7" or update it to "3.4.11" in the composer package.json (which is the version in publish) the composer snapshots fail due to a error related to not finding InternalPusher from 'pusher-js'. Do you have any thoughts on this?

File: __snapshots__/snapshot.test.js.snap [hunk 1]
```diff
@@ -4469,6 +4469,18 @@ exports[`Snapshots CloseAccount showModal 1`] = `
 </section>
 `;
 
+exports[`Snapshots Composer Dashboard (create new) 1`] = `<div />`;
```
Comment (karelm) [inline __snapshots__/snapshot.test.js.snap]: I know I pushed this up, but this doesn't seem quite right to me @anafilipadealmeida

Comment (anafilipadealmeida) [reply __snapshots__/snapshot.test.js.snap]: Maybe we could not include the Composer Snapshots at this point? 🤔We were already not running them, so maybe it's worth revisiting this once the test setup is all done?

Comment (karelm) [reply __snapshots__/snapshot.test.js.snap]: these stories are just the ones in _story.jsx

**Pull request closed without merging**